bin/.gitignore

```diff
@@ -0,0 +1 @@
+ndoe_modules/
```

Comment:
I suggest to put this into the toplevel `.gitignore` with `bin/node_modules` bc its cumbersome to find all igonerfiles scattered around in a project.